### DISCUSSION TITLE
cn/2024/11/14/power-alpha-errors/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#186] [Announcements]
# cn/2024/11/14/power-alpha-errors/

https://hongtaoh.com/cn/2024/11/14/power-alpha-errors/

**Comment by earfanfan** [comment]
好奇问问，你学了这些统计学的知识以后，有没有体会到这门学科的一些观念，比如“为不确定性做确定性的度量”，以及前些时候人们说的“谨防身边统计学”，还有“回归”。当然，我看你写出来的都是概率论、假设检验方面的知识，不知道有没有学到最小二乘法，学到回归、残差等概念。

**Reply by hongtaoh** [reply]
>为不确定性做确定性的度量 

这个我是有体会到的。

>谨防身边统计学

这个我没有什么体会，但是我隐约知道统计学有滥用的情况（比如社会科学和医学）。我现在还不太了解 mediation 和 structural equation modeling 的原理，但我知道模型有人会滥用或者说不正确地诠释其结果，把相关性解释成因果。

下一章就是 ANOVA，最小二乘和残差，快了。我也在学 [贝叶斯统计](https://bayesian.netlify.app/01_prob)。

**Reply by earfanfan** [reply]
我看到你在写方差分析时吐槽自己成为了自己最讨厌的人，只讲知识是什么，不讲知识为什么产生而延续应用至今。我想你也许该明白了，讲是什么和讲为什么本身该是两条路。记得你之前还说过，学习知识不用系统学习，只需要学习需要的部分，可是很多领域知识并不像你学 python 那样被正好地拆成了不同的模块，看上去是相互独立的。许多时候一门学科发展是经历过漫长历史的。

想给你安利两本书，一本是C.R.劳的《统计与真理：怎样运用偶然性》，这本我是2022年看的，很多基础知识都忘光，等于只看了其中偏向于哲学的部分，当然现在更加连书里有过什么观点都忘光了，印象最深刻的只剩下“不确定性知识+所含不确定性度量的知识=可用的知识”，也就是“为不确定性做确定性的度量”。感觉你刚刚学过的知识看这本书可能很合适。而且我也不知道我看的中文本翻译的咋样，你英文好，可以直接看原版了。

第二本是《女士品茶》，这书太长了，翻译得有点絮絮叨叨的，我认真看的就只有最前面和最后面几个章节，中间看的快些。这本你也是可以看英文原版去。这本是我今年才看的，看完以后有些震撼，不过凭我现在的领悟能力还无法描述出来哪里令我感到极为震撼。又不过我摘录微信读书上面一位名为“清和”的读者写的读后感给你瞅瞅，这位写的好。

>我们凭借对世界的观测来认识世界，但如何解释得到的观测值，转化为我们的知识、或帮助我们做决策呢？本书讲述了统计世界观的几次更迭：
>
>（1）17世纪以来，物理学的进展让学者们认为：观测值由理论公式确定，理论与实际的差别是检测手段的问题。但随着测量精度的提高，人们发现了越来越多、无法消除的误差。（2）到了18世纪，拉普拉斯首先提出，误差不是可唯一确定的、由实验操作决定的偏离，而是一种随机变量。我们无法确定某一次实验的具体误差值，但不同误差值出现的概率不同，可以用分布函数描述。（3）19世纪，皮尔逊更进一步，认为观测值本身就是一种随机变量，可以用分布函数描述。而且根据样本，可以计算出分布函数的参数。（4）费希尔再进一步，认为除了观测值，决定观测值分布函数的参数，其本身也是随机变量。（5）由于对概率的解释不同，统计学分裂为两大阵营：频率学派、贝叶斯学派。前者认为概率代表客观事件的发生频率，后者认为概率代表主观个体的认知确定性。（6） 20世纪，混沌理论被提出，认为测量值的随机性只是一种表象，是一个确定公式反复迭代产生的看似随机的结果。虽然这个理论没有被主流接受，却也拥有众多支持者。
>
>现在我们明确了，要用分布函数来描述观测值。但分布函数从哪来？答案是，被统计学家发现的。符合数学要求的分布函数有无数种可能，只有与具体问题能对应起来的函数，才会进入统计学家的工具箱。最重要的分布函数是正态分布，而它重要的原因就是能解释的问题非常多——这一点由中心极限定理保证。书里提到了拉普拉斯的误差分布、皮尔逊的偏斜分布族、戈赛特的t分布、蒂皮特的极值分布、内曼的传染分布等，向读者展示统计学如何一次次增强我们对世界的解释能力。此外，也提到了统计学在具体领域的进展，例如随机过程应用在时间序列，概率单位分析应用在毒理学，风险分析应用在公共政策的决定等。
>
>每个分布函数都有数量不等的参数，它们决定了分布函数的具体形式。如何从观测样本得到符合要求的参数，确定观测总体的分布函数呢？这个过程被称为参数估计。皮尔逊提出了矩估计，费希尔提出了极大似然估计，内曼提出了区间估计，贝叶斯学派使用特有的贝叶斯估计。一些更高级的算法也应用到参数估计中，例如Bootstrap、EM算法、贝叶斯层次模型。
>
>很多时候我们不满足于描述观测，还想比较不同组别观测的差异，明确到底是有本质区别还是巧合。这个过程被称为统计检验。皮尔逊最早提出了卡方检验，应用于离散型观测值的检验。戈赛特提出了t检验，应用于连续型观测值的检验。皮尔逊的儿子埃贡·皮尔逊和内曼，提出零假设和备择假设、显著水平和统计效力等概念，一起完善了统计检验的标准流程，被后世称为假设检验。以上所述，都假定观测服从某种分布，比较由样本估计出的总体（确切来说，总体分布函数的参数，一般指均值）是否有差异，因此称为参数检验。但如果不知道总体服从什么分布呢？那就不假定分布函数，例如Wilcoxon秩和检验，直接根据样本观测值的大小位次判断是否有差异。这种方法称为非参数检验。
>
>大多数情况下，我们也探究因素（自变量）和观测（因变量）之间的关系，但复杂的变量组合往往让这种探究无疾而终。费希尔明确提出，想要探究变量间的关系，必须依赖于实验的合理设计。他用P值描述自变量是否对因变量产生影响，用多因素方差分析、协方差分析分离不同自变量的效果。费希尔强调，如果不使用随机实验设计，就证明不了任何结论。但是，现实有时不允许随机，尤其涉及到人类健康时，随机实验面临严重的伦理问题。于是，病例对照研究、回顾性研究、观察性研究、意向性治疗等研究手段被提出。并非完全随机的实验设计，导向了有前提的实验结论，但总比没有结论好。
>
>细心的朋友可能发现了，P值不是用于在假设检验中判断是否拒绝原假设吗，怎么去描述自变量影响了？其实这里大有渊源。为了检验某个自变量是否对因变量有影响，费希尔提出了P值。在费希尔这里没有零假设、备择假设之分，也没有划定一个“显著影响”的界限，依靠P值的直观大小衡量影响程度。被后世称为“显著性检验”。为了比较两组样本是否有差异，内曼、埃贡·皮尔逊提出了显著性水平α。他们的推断体系里存在零假设、备择假设，推荐的显著性水平是0.05，依靠构造统计量与α比较来得出结论。这个体系没提到P值。被后世称为“假设检验”。三人互相争论很久，也没达成一致。后来在实践中，费希尔的P值计算简单，内曼-皮尔逊的检验流程更标准化，就被杂糅到一起，成为现在假设检验的样子。
>
>统计学的革命固然波澜壮阔，但统计学的基础却远非牢固。现在人们时常讨论概率、相关，却并不清楚自己在说什么。因果关系被认为理所当然，但数学上的证明、哲学上的确立都难如登天。最需要调用理性、靠数据说话的学科，却触碰到了人类理性能掌握的极限，真是奇特的现象。
>
>非统计学/数学专业，难免存在谬误，且挂一漏万。请诸君批评指正

**Reply by earfanfan** [reply]
>我隐约知道统计学有滥用的情况（比如社会科学和医学）。我现在还不太了解 mediation 和 structural equation modeling 的原理，但我知道模型有人会滥用或者说不正确地诠释其结果，把相关性解释成因果。

mediation 这个是撒我不知道，structural equation modeli 这个翻译过来就是结构方程模型吧。模型的滥用，大概是人类社会里一种必然出现的现象，因为牵涉到了利益……统计学是一种解释数据的工具，被人类拿去用的时候，在很多别的方面还会被出于各种目的而使用，常常是显得某某而实际经不起推敲，国内吐槽比较多的就是每次平均工资的统计数据一出来，大家就吐槽自己被平均了，或者吐槽自己成为了某些指标的分母，以及在使用文本描述指标数据的方面，还有“负增长”之类极具创造性的词语，还有一些抽样调查。

**Reply by hongtaoh** [reply]
嗯嗯，是结构方程模型。你提到的这些更多的是「描述统计」层面的，这个大多数人还可以理解。但是一个人滥用结构方程模型，如果不是专家，根本挑不出错来。前段时间有个正规期刊让我审稿，我在摘要里看到结构方程模型几个字就直接拒绝了审稿邀请，因为我不懂它，无法评价。你提到的「利益」，确实是的。为了发表，就需要显著，不知道那些统计大师看到自己的发明被如此用会怎么想。

**Reply by earfanfan** [reply]
有些模型的滥用和工具的易用是相伴相生的，我记得用来算结构方程模型各种参数的工具大概叫做 AMOS。我读研的时候同宿舍有个女孩就是用结构方程模型写的毕业论文，她的导师带的好几届学生全都是这样，做点抽样调查搞点数据就能写论文了。仔细想想，我的毕业论文也是滥用了模型的，用的是 SVAR，尽管我当时费劲收集数据、学习这个模型，又看了好多好多别人的论文，不过也只是走了一个很标准的写论文的流程而已，写出来的东西实在是太烂了，惭愧呀。唉，不过，不过我昨天还精神抖擞可以噼里啪啦写好多，今天就感觉要狗带，主要是昨晚睡落枕了，哎，感觉不太行，精气神都冇了，撤了，拜拜，以后再来叨扰。

**Reply by hongtaoh** [reply]
我对这些模型不懂，所以就不评价它们本身。我只是觉得，那玩意就跟一个黑箱一样，我猜 99% 用它的人根本不知道这些模型的原理。别说 SEM 这种模型，就连 ANOVA，十个用的人里面能有两个彻底搞懂原理都已经超出我的认知了。最后大家都不懂装懂，模型越弄越复杂，最后的结论越来越难以评估。

**Reply by earfanfan** [reply]
额，你这个学计算机的糊涂男银，你说的是不是 ANOVA，即方差分析。使用微观数据做分析的时候，单因素方差分析应该还不至于搞不懂。统计学算是一门交叉学科，很多其他领域的学生都会学点统计学基础的。我记得以前忘了谁给我讲过，大学的时候就有一门基础课，名为统计学。

>最后大家都不懂装懂，模型越弄越复杂，最后的结论越来越难以评估。

我倒觉得说不定是为了发论文为了毕业，太基础的方法已经被卷爆了，所以就开始卷那些可解释性越来越差的模型了。正好我最近打算看看我国的教育统计数据，放点出来给你瞅瞅，2004年的时候，分学历的招生数是，博士5.3万，硕士27.3万，普通本科209.9万，普通专科237.4万，到2022年的时候变成了博士13.9万，硕士110.4万，普通本科467.9万，普通专科538.9万。把这个招生数量转换成论文数量……

**Reply by earfanfan** [reply]
@hongtaoh 
Dear 郝糊涂：
<https://yanyuteng.github.io/2024/06/15/a6894eaa/>这里的评论区你有空可以去溜达一下，我和一位同学@yanyuteng 也交流了一下模型滥用的问题，当然啦，我只是一块普通的砖头，你俩才是玉，虽然大家都是石头，哈哈。

**Reply by hongtaoh** [reply]
>你说的是不是 ANOVA，即方差分析。

我 *。我居然打错了。。

>使用微观数据做分析的时候，单因素方差分析应该还不至于搞不懂

了解了！那是我夸大了。

>太基础的方法已经被卷爆了，所以就开始卷那些可解释性越来越差的模型了

嗯，没啥好说的，大家都是混口饭吃。

**Reply by hongtaoh** [reply]
@yanyuteng 这哥们的统计我没看明白，初恋日记倒是看馋了。

**Reply by earfanfan** [reply]
>……初恋日记……

瞬间竖起俺关注八卦的天线……

嗷，不对呀，按照咱俩以前讨论出来的找对象方法论，第一步是多参加活动勇于搭讪，第二步是释放自己的魅力，第三步是自然水到渠成。我觉得我找不到对象很正常，我都不敢去搭讪的，前些天坐高铁回来一边坐一个大帅哥我都没好意思多看几眼。但你对搭讪是很敢的啊，我以为你找对象很容易呢。

**Reply by hongtaoh** [reply]
看来你修炼得不错。帅哥都可以不看。

这个三部方法论在我这里不适用，因为我整天都窝在家里做科研。说到搭讪，我现在刻意让自己不去搭讪，因为没有什么结果。这个三步方法论对我来说在中国适用，在北美就不适用。

**Reply by yanyuteng** [reply]
哈哈哈哈是俺师弟的八卦 ~~还是八卦有趣~~
